src/main/webapp/app/shared/metis/posting-reactions-bar/posting-reactions-bar.component.ts

```diff
@@ -11,6 +11,10 @@ const PIN_EMOJI_ID = 'pushpin';
 const PIN_EMOJI_UNICODE = '1F4CC';
 const ARCHIVE_EMOJI_ID = 'open_file_folder';
 const ARCHIVE_EMOJI_UNICODE = '1F4C2';
+const SPEECH_BALOON_ID = 'speech_balloon';
+const SPEECH_BALOON_UNICODE = '1F4AC';
+const HEAVY_MULTIPLICATION_ID = 'heavy_multiplication_x';
```

Comment:
Considering our discussion about the visibility of these emojis in dark mode you should reconsider the usage of this emoji in the UI.
It would be a better idea to use a Font Awesome icon like `fa-xmark` instead. This would also have the benefit of having the same color as the text.

Reply:
I actually tried it with fa-conversation icon first and did not like the overall look. I think the "integration" of the button within the emoji-bar should actually use a real emoji. BTW, the different color is also present for other emojis at the moment while we display the number of times en emoji is selected.